Fixed outdated code and documentation in A-Frame tutorial in Games section (#34159)
mdn/content: files/en-us/games/techniques/3d_on_the_web/building_up_a_basic_demo_with_a-frame/index.md

## High level overview

> The current version of A-Frame is 1.6.0, which means it's highly experimental, but it already works and you can test it right away in the browser. It runs on desktop, mobile (iOS and Android), and Oculus Rift, Gear VR and HTC Vive.

Reviewer: The version info will get outdated quickly, so we can remove mention of it and rework this to say it runs everywhere.

```suggestion
A-Frame runs on most environments, such as desktop, mobile (iOS and Android), and devices such as Oculus Rift, Gear VR and HTC Vive.
```

Author: According to me that sounds perfect, There is also another issue. That the solution code presented on this doc, uses the outdated code. I already created a pull request , at the original directory of the solution, but it will probably take some time till somebody reviews it.

## Animation

We've already used `rotation` and `position` to move the shapes on the scene, and we can also scale them. These attributes can be manipulated to create the illusion of [animation](https://aframe.io/docs/1.6.0/components/animation.html).

### Rotation

There's a special [`animation`](https://aframe.io/docs/1.6.0/components/animation.html) component that can help us animate elements. Add the `animation` component seen below to the `<a-box>` element as a property, as shown:

```html
<a-box
  color="#0095DD"
  rotation="20 40 0"
  position="0 1 0"

>   animation="property: rotation;from:20 0 0 to: 20 360 0;dir: alternate; loop: true; dur: 4000; easing: easeInOutQuad;">

Reviewer: I haven't verified, but is it supposed to be this syntax?

```suggestion
  animation="property: rotation; from: 20 0 0; to: 20 360 0; dir: alternate; loop: true; dur: 4000; easing: easeInOutQuad;">
```

Author: @bsmth. Yes the semicolon needs to be there, cause according to the docunmentation to and from are distinct properties. If you check the [docs](https://aframe.io/docs/1.6.0/components/animation.html#direct-values-through-object3d-and-components), there is also a small example.

Reviewer: Super, thank you. I'll take a proper look through shortly. I can see there's some other problems with the page (we're talking about [older versions](https://github.com/mdn/content/pull/34159/files#diff-bf5f9fb5c4a9cae88af6376dc5b20d7945104d72fa957eb2d8d828ee41658571R13) in the intro). And it might be nice to remove the jsfiddles somehow.

Author: Thank you as well 😊 . The current stable version seems to be 1.6.0. What if we tried to update the code in the jsfiddles? At least the part of it that's changed?

Author: I replaced the jsfiddle, with a working example of it that uses the updated code.

Author: @bsmth can you check this new changes?

Everything is rendered properly and animating — congratulations on building your first A-Frame scene! Here's how the final version looks and works:

> {{JSFiddleEmbed("https//jsfiddle.net/lowperry/xmo62ku0/5/embed/","","350")}}

Reviewer: Minor fix:

```suggestion
{{JSFiddleEmbed("https://jsfiddle.net/lowperry/xmo62ku0/5/","","350")}}
```
